model/CMakeLists.txt

```diff
@@ -19,6 +19,8 @@ set(sail_common
     --strict-exponentials
     # Minimum required Sail compiler version.
     --require-version ${SAIL_REQUIRED_VER}
+    # Put Sail smt cache in /build directory since by default Sail will
```

Comment:
```suggestion
    # Put Sail smt cache in /build directory since by default Sail will
    # place it in its working directory which is ${CMAKE_CURRENT_SOURCE_DIR}.
```